morph/cli.py

```diff
@@ -139,14 +168,23 @@ def main():
     p_dump.add_argument('name', metavar='NAME', help='database to dump (all, known, ...)')
     p_dump.add_argument('--freq', action='store_true', help='include frequency as known to MorphMan')
 
+    def add_mizer(parser):
+        parser.add_argument('--mizer', default='mecab', choices=MIZERS.keys(),
+                            metavar='NAME',
+                            help='how to split morphemes (%s) (default %s)'
+                                 % (', '.join(MIZERS.keys()), 'mecab'))
+
     p_count = subparsers.add_parser('count', help='count morphemes in a corpus',
                         description='Count all morphemes in the given files and emit a frequency table.')
     p_count.set_defaults(action=cmd_count)
     p_count.add_argument('files', nargs='*', metavar='FILE', help='input files of text to morphemize')
-    p_count.add_argument('--mizer', default='mecab', choices=MIZERS.keys(),
-                         metavar='NAME',
-                         help='how to split morphemes (%s) (default %s)'
-                              % (', '.join(MIZERS.keys()), 'mecab'))
+    add_mizer(p_count)
+
+    p_next = subparsers.add_parser('next', help='find next notes to study from a corpus')
+    p_next.set_defaults(action=cmd_next)
+    p_next.add_argument('prio', metavar='FREQS', help='file of morphemes to study, with frequencies')
+    p_next.add_argument('notes', metavar='NOTES', help='file of newline-terminated notes, each tab-separated fields starting with the text')
```

Comment:
> In any event the format should be documented. This is the same format emitted by mm dump -- perhaps a good way to document it would be to explain in that command's help, and refer briefly to mm dump --help from wherever else it comes up?

Yes, that makes the usage much clearer.